lib/active_merchant/billing/gateways/rapyd.rb

```diff
@@ -203,13 +203,19 @@ def add_ewallet(post, options)
       def add_payment_fields(post, options)
         post[:payment] = {}
 
-        post[:payment][:description] = options[:description] if options[:description]
-        post[:payment][:statement_descriptor] = options[:statement_descriptor] if options[:statement_descriptor]
+        post[:description] = options[:description] if options[:description]
+        post[:statement_descriptor] = options[:statement_descriptor] if options[:statement_descriptor]
```

Comment:
If we're not adding it to `post[:payment]` then let's not set `post[:payment] = {}`

Reply:
Ah, good catch. I will take that out.